android/build.sh

```diff
@@ -37,6 +37,10 @@ builder_describe \
 
 builder_parse "$@"
 
+# Override JAVA_HOME to OpenJDK 11
+builder_heading "Setting JAVA_HOME to OpenJDK 11"
+export JAVA_HOME=${JAVA_HOME_11}
```

Comment:
So, a problem here:  if the build fails, I doubt that `JAVA_HOME` will be restored properly given the current implementation of this build script.

It may be wise to extend the existing builder trap (because only one may be set...) and add functionality there to ensure that `JAVA_HOME` is fixed before re-throwing an exit-triggering exit code.

For the in-repo reference, refer to `_builder_failure_trap` via CTRL-F.  It'd probably be best to simply write your own trap and have it call this one, and then replace this one with your version.

Reply:
`JAVA_HOME` doesn't need to be reset -- you don't modify the environment for the caller of the script with `export`, only with child processes. (You can only modify 'parent' environment with `source`d scripts -- which run in the same process not a child process.)

Reply:
Ah, good to hear.  I inferred that it did based on the implementation I reviewed, but didn't investigate it with any serious depth.  Glad to hear it's not actually going to be an issue.